pt-BR/mail/chrome/messenger/am-main.dtd

```diff
@@ -2,7 +2,7 @@
 
 <!ENTITY accountTitle.label "Configurações da conta">
 
-<!ENTITY accountNameDesc.label "The account name is used to identify each account">
+<!ENTITY accountNameDesc.label "O nome da conta é usado para identificar cada conta">
 <!ENTITY name.label "Nome:">
 <!ENTITY name.accesskey "N">
 <!ENTITY email.label "Email:">
```

Comment:
@brunotiussi I understand your position, however my recommendation is to translate all possible words to PT-BR. If you'd like to keep it short (GUI purposes) either use e-mail or email (chose one to keep consistency).